src/main/java/com/google/sps/servlets/BusinessServlet.java

```diff
@@ -15,44 +15,43 @@
 public class BusinessServlet extends HttpServlet {
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-      String bus= csvToJson("/home/skolb/software-product-sprint/team13/src/main/webapp/businesses.csv");
-      response.setContentType("application/json;");
-      response.getWriter().println(bus);
+    String bus =
+        csvToJson("/home/skolb/software-product-sprint/team13/src/main/webapp/businesses.csv");
```

Comment:
Does the absolute path work on the deployed site?

Reply:
No it doesn't, I'm trying some changes and it's not working

Reply:
fixed it!